wlm/src/main/java/ai/djl/serving/wlm/LmiConfigRecommender.java

```diff
@@ -46,6 +51,9 @@ public final class LmiConfigRecommender {
                     Map.entry("qwen2", "vllm"),
                     Map.entry("stablelm", "vllm"));
 
+    private static final Set<String> OPTIMIZED_TASK_ARCHITECTURES =
+            Set.of("ForCausalLM", "LMHeadModel", "ForConditionalGeneration");
```

Comment:
https://huggingface.co/THUDM/chatglm3-6b/blob/main/config.json#L5 this is going to break

Reply:
ugh, that's good to know. didn't know about this use-case

Reply:
i've added auto_map to fix this use-case